Sources/SwiftQueue/SqOperationQueue.swift

```diff
@@ -136,6 +136,14 @@ internal final class SqOperationQueue: OperationQueue {
             operation.cancel()
         }
     }
+    
+    func operationCount(tag: String) -> Int {
+        return operations.compactMap({ $0 as? SqOperation }).filter({ $0.info.tags.contains(tag) }).count
+    }
+    
+    func operationCount(uuid: String) -> Int {
+        return operations.compactMap({ $0 as? SqOperation }).filter({ $0.info.uuid == uuid }).count
```

Comment:
To check if a job exist I would rather do it in a fully swift way than with High order function. Should be more optimised

```swift
    public func containsJob(uuid: String) -> Bool {
        for element: OperationQueue in manage.values {
            for case let item as SqOperation in element.operations where item.info.uuid == uuid {
                return true
            }
        }
        return false
    }
```

Reply:
I'll update that